### ISSUE TITLE
Setting end effector constraints when using STOMP

### ISSUE THREAD

**Issue by ghost** [ros-industrial-attic/industrial_moveit#68]
Hi all,
I'm trying out STOMP as per the tutorial [here](http://ros-industrial.github.io/industrial_training/_source/session5/Introduction-to-STOMP.html), but havent figured out how to set orientation constraints for the end effector so that it maintains a certain orientation throughout the trajectory. Is it possible to set this via yaml?

**Comment by Levi-Armstrong** [comment]
I don't believe this is currently implemented. You will need to write a noisy generator plugin that generator states that satisfy your constraints. You could get this from moveit ompl  constraint sampler. Then may need a custom plugin for merging the noisy trajectories but not certain this plugin would be needed.

**Comment by ghost** [comment]
Thank you for the reply Levi. Would this [link](http://docs.ros.org/indigo/api/moveit_tutorials/html/doc/custom_constraint_samplers.html) be a good starting point?

**Comment by jrgnicho** [comment]
This issue is partially addressed [now](https://github.com/ros-industrial/industrial_moveit/pull/46) in that you can set orientation constraints on the cartesian goal however STOMP won't apply this to the rest of the waypoints in the trajectory.

**Comment by ib101** [comment]
Any update on this? Would love to use STOMP, its STOMP-tastic!

**Comment by jrgnicho** [comment]
Hello @ib101 , the stomp planner packages have been [moved](https://github.com/ros-industrial/stomp_ros) into their own repository.  As far as I know, no one at the moment is working on such a feature but you're welcome to submit a PR for this in the new repo.
